### PR TITLE
Fix debugger and port forwards in devcontainer

### PR DESCRIPTION
**Pull request by felipecrs**
## Breaking change

N/A

## Proposed change

As suggested by @frenck at [this comment](https://github.com/home-assistant/frontend/pull/14891#discussion_r1057831813).

The debugger configuration wasn't working for me. I suppose something changed in the VSCode/the debugger/webpack since then. After spending some time on it, I realized that the default configuration for webRoot works while the one previously set here doesn't, so I simply removed it. Renamed pwa-chrome to chrome as the former was deprecated. I did not test the debug configuration for anything but Frontend (because the build task wasn't working for me and I didn't want to spend more time on it).

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -4,7 +4,25 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": "8124:8123",
+  "forwardPorts": [8123, 8100, 8090, 8080],
+  "portsAttributes": {
+    "8123": {
+      "label": "Home Assistant",
+      "onAutoForward": "silent"
```
Comment (ludeeus) [inline .devcontainer/devcontainer.json:11]: Why is "silent" added here (and for the others)?
The default is "notify" which is quite nice.

Comment (felipecrs) [reply .devcontainer/devcontainer.json:11]: It's just my preference, I'll change it. I should work on this PR tonight.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -4,7 +4,21 @@
     "dockerfile": "Dockerfile",
     "context": ".."
   },
-  "appPort": "8124:8123",
+  "forwardPorts": [8123, 8100, 8090, 8080],
+  "portsAttributes": {
+    "8123": {
```
Comment (ludeeus) [inline .devcontainer/devcontainer.json:9]:
```suggestion
  "forwardPorts": [
    "8124:8123",
    8100,
    8090,
    8080
  ],
  "portsAttributes": {
    "8124:8123": {
```
Lets keep the current mapping to not conflict with the core devcontainer.

**Review by ludeeus — DISMISSED** [review]

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (felipecrs) [reply .devcontainer/devcontainer.json:9]: Not needed. The way I had set it will tell vscode to use 8123 if possible, otherwise the next available port.

Comment (ludeeus) [reply .devcontainer/devcontainer.json:9]: I tested it, it's definitely needed.

Comment (felipecrs) [reply .devcontainer/devcontainer.json:9]: @ludeeus that's most likely because your are not using https://github.com/home-assistant/core/pull/82753.

Comment (ludeeus) [reply .devcontainer/devcontainer.json:9]: No, I'm using dev.

Comment (felipecrs) [reply .devcontainer/devcontainer.json:9]: Yes, so let's try to merge both. I'll fix the needed stuff in https://github.com/home-assistant/core/pull/82753.

**Comment by ludeeus** [comment]
Marking as draft as merging will break current setups.

**Pull request closed without merging**